docs/migration.md

```diff
@@ -13,6 +13,52 @@ You'll need to update any `import`/`require` statements in your support code to
 
 (The executable is still `cucumber-js` though.)
 
+## Hooks
+
+The result object passed as the argument to your `After` hook function has a different structure.
+
+Previously in `cucumber`:
+
+```js
+{
+  "sourceLocation": {
+    "uri": "features/example.feature",
+    "line": 7
+  },
+  "pickle": {...},
+  "result": {
+    "duration": 660000000,
+    "status": "failed",
+    "exception": {
+      "name": "AssertionError",
+      "message": "...",
+      "showDiff": false,
+      "stack": "..."
+    },
+    "retried": true
+  }
+}
+```
+
+Now in `@cucumber/cucumber`:
+
+```js
+{
+  "gherkinDocument": {...}, // schema: https://github.com/cucumber/common/blob/main/messages/jsonschema/GherkinDocument.json
+  "pickle": {...}, // schema: https://github.com/cucumber/common/blob/main/messages/jsonschema/Pickle.json
```

Comment:
As a note, linking to a branch has the potential of going stale. May be better to update these to a tag